### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -14016,6 +14021,9 @@ reesrved->reserved
 reesult->result
 reevalute->reevaluate
 reevaulating->reevaluating
+refartor->refactor, refractor,
+refartored->refactored, refactored,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Did you mean refracted? It's currently a duplicate word as per Travis.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Ooops :blush:  
Thanks

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 8a48688

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -18821,6 +18829,7 @@ visious->vicious
 visisble->visible
 visiter->visitor
 visiters->visitors
+visivble->visible
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Could add invisible version too.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Good idea. Will add to my next PR.

**Pull request merged**